docs/SBVR.md

```diff
@@ -0,0 +1,355 @@
+# SBVR in Pine.js
+
+An SBVR model is composed of one or more _vocabularies_, serving as namespaces. Each vocabulary is composed of _terms_, _fact types_, and _rules_, which together define the data model and its constraints. Terms from different vocabularies can be referenced adding the name of the vocabulary, in parenthesis, after the term.
+
+SBVR files are composed of statements, one per line, in the form `<header>: <body>`. Indentation is not significant, and `--` introduces comments that go until the end of the line. SBVR files accepted by Pine.js are case-insensitive and usually have the `.sbvr` extension. `Vocabulary`, `Term`, `Fact Type`, and `Rule` are toplevel declarations that can be followed by _attributes_. See the **Attributes** subsection below.
+
+A vocabulary can be started with:
+
+```
+Vocabulary: <vocabulary-name>
+```
+
+Every new definition is part of the last declared vocabulary. If no vocabulary has been explicitly declared, a `Default` vocabulary is implicitly assumed. Within vocabularies, term declarations introduces _types_:
```

Comment:
This could be expanded a bit here: in what sense is a term a type?

Reply:
Here's the explanation I put:

Terms are _types_: they represent a collection of instances that obey an uniform "interface". And that is defined through fact types and rules.